### ISSUE TITLE
Email Reports Admin Page: Incorrect use of article landmark

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1635]
/wp-admin/admin.php?page=accessibility_checker_settings&tab=accessibility-reports uses the article landmark for each section on the page:

<img width="1521" height="553" alt="Image" src="https://github.com/user-attachments/assets/79165c65-0f38-46a8-9986-8b45d1a7cf57" />

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/article):
>The `<article>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). Examples include: a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, an interactive widget or gadget, or any other independent item of content. 

This does not meet that criteria and should not use the article tag.
